src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -425,15 +429,15 @@ public void logDocumentRead(String index, String id, ShardId shardId, Map<String
     @Override
     public void logDocumentWritten(ShardId shardId, GetResult originalResult, Index currentIndex, IndexResult result) {
 
-        if(complianceConfig == null || !complianceConfig.writeHistoryEnabledForIndex(shardId.getIndexName())) {
+        if(complianceConfig == null || !complianceConfig.isEnabled() || !complianceConfig.writeHistoryEnabledForIndex(shardId.getIndexName())) {
```

Comment:
Saw that this was used twice. May want to make this into a common utility function.